RFunction.R

```diff
@@ -489,8 +508,20 @@ rFunction = function(data, travelcut,
   
   
 
-  #### 4. Roosting Classification -----
-  logger.info("[4] Performing roosting classification")
+  ## [4] Roosting-site Classification -------------
+  
+  #' Remaining resting locations re-classified as roosting if identified as part
+  #' of a roosting-site, which is defined as: 
+  #' 
+  #' Consecutive stationary locations (`roostgroup`) encompassing night-time
+  #' locations with total overnight distance traveled less than 15 meters
+  #' (`roostsite`)
+  #' 
+  #' NOTE: STravelling locations not affected by this step, even if they were
+  #' tagged as part of a roost-site
```

Comment:
for example, the last location at a roost site will be given the behaviour of travelling as we look ahead to the next point?

Reply:
Yes, I think that would be the case